Cavdy/Banka
# #165518021 Get transaction by account number and transaction id

#### What does this PR do?
Get transaction by account number and transaction id

#### Description of Task to be completed?
- Log in as a user and navigate to the transactions you should see your transactions if you have any

#### How should this be manually tested?
- Log in as a user and navigate to the transactions you should see your transactions if you have any

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165518021](https://www.pivotaltracker.com/story/show/165518021)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
